src/db/src/db/data/seeds/001favorites.js

```diff
@@ -0,0 +1,10 @@
+exports.seed = async function(knex) {
+  await knex('favorite').del();
+
+  await knex('favorite').insert([
+    { user_id: 1, recipe_id: 'spaghetti_bolognese', region: 'Emilia-Romagna' },
+    { user_id: 2, recipe_id: 'carbonara', region: 'Lazio' }
+  ]);
+};
```

Comment:
😲 Oh! You made a database and inserted data, too. Why did you ask me to work on the database if you planned to do it anyway? I asked you to talk about how to seed the database. You even approved my pull request. So, I am very confused.

I think we may have some communication problems in our group that we should address in the morning. Right now, I feel that all the work I have done yesterday afternoon and today has not provided a single contribution to our group.